numba/cuda/tests/cudapy/test_atomics.py

```diff
@@ -715,6 +795,168 @@ def test_atomic_xor_global_2(self):
         cuda_func[1, (4, 8)](ary, rand_const)
         np.testing.assert_equal(ary, orig ^ rand_const)
 
+    def test_atomic_inc(self):
+        rand_const = np.random.randint(32)
+        ary = np.random.randint(0, 32, size=32).astype(np.uint32)
+        idx = np.arange(32, dtype=np.uint32)
+        orig = ary.copy()
+        cuda_func = cuda.jit('void(uint32[:], uint32[:], uint32)')(atomic_inc)
+        cuda_func[1, 32](ary, idx, rand_const)
+
+        self.assertTrue(np.all(ary == np.where(orig >= rand_const, 0,
+                                               orig + 1)))
```

Comment:
All these `assertTrue` instances should be replaced to use `np.testing.assert_equal` instead, because it produces better diagnostics when the test fails. E.g.:

```suggestion
        np.testing.assert_equal(ary, np.where(orig >= rand_const, 0, orig + 1))
```